modules/movies/api/movies-routes.ts

```diff
@@ -0,0 +1,68 @@
+import * as core from 'express-serve-static-core';
+import getMovieById from '../utils/getMovieById';
+import getRandomMovies from '../utils/getRandomMovies';
+
+module.exports = function (app: core.Express) {
+    /**
+     * @openapi
+     * '/api/movies/':
+     *  get:
+     *    tags:
+     *    - Movies
+     *    summary: Returns a random array of ids
+     *    parameters:
+     *    - in: query
+     *      name: qty
+     *      schema:
+     *          type: number
+     *          default: 10
+     *    responses:
+     *      '200':
+     *        description: OK
+     *        schema:
+     *          type: array
+     *          items:
+     *            $ref: '#/definitions/MockMovie'
+     */
+    app.get('/movies', (req, res) => {
+        const { qty = 10 } = req.query;
```

Comment:
Hi we have a util for getting the qty from the route, getCountryNameFromRequest(req);
also your route does not follow our pattern eg. '/address/:country?/:qty?'

Reply:
Hi, sure. The changes have been made